concepts/tuples/about.md

```diff
@@ -4,37 +4,33 @@ A tuple is a versatile and efficient compound data type in Cairo, allowing you t
 
 ## Creating Tuples
 
-To create a tuple, you write a comma-separated list of values inside parentheses. Each position in the tuple has a specific type, and these types can differ from one another. Here’s an example with optional type annotations:
+To create a tuple, you write a comma-separated list of values inside parentheses. Each position in the tuple has a specific type, and these types can differ from one another.
 
 ```rust
 fn main() {
     let tup: (u32, u64, bool) = (10, 20, true);
 }
 ```
 
-In this example, the variable `tup` binds to the entire tuple, treating it as a single compound element.
-
 ## Destructuring Tuples
 
-To access the individual values within a tuple, you can use pattern matching to destructure the tuple. This breaks the tuple into its constituent parts. For example:
+To access the individual values within a tuple, you can use pattern matching to destructure the tuple. This breaks the tuple into its constituent parts.
 
 ```rust
 fn main() {
     let tup = (500, 6, true);
 
     let (x, y, z) = tup;
 
-    if y == 6 {
-        println!("y is 6!");
-    }
+    println!("x is: ", {x}); // prints "x is: 500"
```

Comment:
```suggestion
    println!("x is: {x}"); // prints "x is: 500"
```
update the other printlns the same way, the current code will not compile due to wrong syntax